<launch>
  <!--  smooths inputs from cmd_vel_mux/input/teleop_raw to cmd_vel_mux/input/teleop -->
  <include file="$(find teleop_turtlebot)/launch/includes/velocity_smoother.launch.xml"/>

  <node pkg="teleop_turtlebot" type="teleop_turtlebot_joy" name="teleop_turtlebot_joystick" output="screen">
    <param name="scale_angular" value="1.5"/>
    <param name="scale_linear" value="0.5"/>
    <param name="axis_deadman" value="7"/>
    <param name="axis_linear" value="1"/>
    <param name="axis_angular" value="0"/>
    <remap from="teleop_turtlebot_joystick/cmd_vel" to="teleop_velocity_smoother/raw_cmd_vel"/>
  </node>

  <node pkg="joy" type="joy_node" name="joystick" />

</launch>
